cdc/scheduler/internal/tp/replication_set.go

```diff
@@ -573,6 +576,7 @@ func (r *ReplicationSet) handleRemoveTable() ([]*schedulepb.Message, error) {
 			zap.Any("replicationSet", r), zap.Int64("tableID", r.TableID))
 		return nil, nil
 	}
+	// todo: OldState must be `replicating` here
```

Comment:
How do we deal with these todo?

Reply:
It looks `OldState` is only used for logging, we can remove it from the log, since we know that it's must be in desired state.